cmd/cosign/cli/verify/verify.go

```diff
@@ -142,7 +142,7 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 		co.TSACerts = tsaCertPool
 	}
 
-	if keylessVerification(c.KeyRef, c.Sk) {
+	if keylessVerification(c.KeyRef, c.Sk) || !c.SkipTlogVerify {
```

Comment:
I *think* these || statements could be replaced by just !c.SkipTlogVerify. I think you know that and will do in a follow up :)

Reply:
Done in https://github.com/sigstore/cosign/pull/2505